src/test/java/gmail/mishchuk/lesson38_Map/MapUtilsTest.java

```diff
@@ -0,0 +1,33 @@
+package gmail.mishchuk.lesson38_Map;
+
+import org.junit.jupiter.api.Test;
+import java.util.*;
+
+import static gmail.mishchuk.lesson38_Map.MapUtils.getUnitedMap;
+import static org.junit.jupiter.api.Assertions.*;
+
+class MapUtilsTest {
+
+    @Test
+    void couldDoReturnUnitedMapFromTwoMaps() {
```

Comment:
Якщо перекласти Ваш тест дослівно то вийде `мігЗробитиПовернутиОб'єднануМапуЗДвохМап`

Краще написати просто `shouldReturnUnitedMapFromTwoMaps`

```suggestion
    void shouldReturnUnitedMapFromTwoMaps() {
```